app/components/govuk_component/cookie_banner_component.rb

```diff
@@ -1,13 +1,14 @@
 class GovukComponent::CookieBannerComponent < GovukComponent::Base
   renders_many :messages, GovukComponent::CookieBannerComponent::MessageComponent
 
-  attr_accessor :aria_label, :hidden
+  attr_accessor :aria_label, :hidden, :hide_when_printing
 
-  def initialize(aria_label: "Cookie banner", hidden: false, classes: [], html_attributes: {})
+  def initialize(aria_label: "Cookie banner", hidden: false, hide_when_printing: true, classes: [], html_attributes: {})
```

Comment:
I'm not convinced `hide_when_printing` is the nicest variable name here. 

I went with `hide_when_printing: true` over `display_when_printing: false` because it made assigning the class using `class_names` more readable.

Reply:
I just checked and there's a similar option in the breadcrumbs component called `hide_in_print` - I think it'd make sense for them to be consistent.

Sorry, didn't realise until now!